mulog-kinesis/src/com/brunobonacci/mulog/publishers/aws_utils.clj

```diff
@@ -0,0 +1,25 @@
+(ns com.brunobonacci.mulog.publishers.aws-utils
+  (:require
+    [cognitect.aws.client.api :as aws]
+    [cognitect.aws.credentials :as credentials]
+    [cognitect.aws.util :as u])
+  (:import (java.nio ByteBuffer)))
+
+(def region (u/getenv "AWS_REGION"))
+
+(def kinesis (aws/client {:api                  :kinesis
+                          :region               (keyword region)
+                          :credentials-provider (credentials/environment-credentials-provider)}))
```

Comment:
This is evaluated at compile time, I think it should be evaluated when the publisher is instantiated
For the region, there is a Default Region provider as well, maybe better to use that one

Reply:
agree, fixed